.github/workflows/build_docs.yaml

```diff
@@ -0,0 +1,36 @@
+# Triggers the workflows on push or pull request events but only for the main branch
+name: Generate new docs.
+on:
+  # Allows you to run this workflows manually from the Actions tab
+  workflow_dispatch:
+
+# A workflows run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflows contains a single job called "build"
+  build:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Runs a set of commands using the runners shell
+      - uses: actions/checkout@v2
+        with:
+          ref: master
```

Comment:
Not sure I understand why this is here 🤔. I think it would be assumed that the docs would be built from the branch running the workflow.